Controller/PlatformUIController.php

```diff
@@ -33,6 +42,28 @@ public function shellAction()
         );
     }
 
+    /**
+     * Load JS or CSS assets.
+     *
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function yuiCombineLoaderAction(Request $request)
```

Comment:
I was thinking about it. I decided to leave it here, because shell action has similar purpose (admin UI loading).

But no problem for me, I can move this to separate file.